CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ You can find all deprecations in [this overview](https://github.com/kedacore/ked
 New deprecation(s):
 
 - **Azure Data Explorer**: Deprecate `metadata.clientSecret` ([#4514](https://github.com/kedacore/keda/issues/4514))
+- **Webhook command**: Removing CLI flag `--tls-min-version` (discussion: [#4582](https://github.com/kedacore/keda/pull/4582#discussion_r1209547634))
```

Comment:
Do we need this @zroubalik @tomkerkhove ? This parameter wasn't documented anywhere and either supported in helm

Reply:
yeah, just drop it, to avoid confusion. We can just mention it in https://github.com/kedacore/keda/issues/4581